code/modules/mob/living/carbon/human/life.dm

```diff
@@ -319,8 +319,16 @@
 	// Tissues die without blood circulation
 	adjustBruteLoss(2)
 
-
-
+/mob/living/carbon/human/handle_thirst()
+	..()
+	if(thirst >= THIRST_LEVEL_OVERHYDRATED)
+		if(prob(5) && w_uniform)
+			Stun(4 SECONDS)
+			visible_message("[src] pees [p_their()] pants!")
+			playsound(src, 'sound/effects/splat.ogg', 50, 1)
```

Comment:
....I don't think it would be this loud.

Wetting the tile or adding a slippyery water-decal would make more sense than this and laughing....but really, this whole mechanic is just...kinda freaking weird.

Reply:
If it wet the tile it would be fetish material

Reply:
\>